src/components/Firebase/firebase.js

```diff
@@ -17,13 +18,17 @@ class Firebase {
 
     /* Helper */
 
-    this.serverValue = app.database.ServerValue;
+    this.fieldValue = app.firestore.FieldValue;
     this.emailAuthProvider = app.auth.EmailAuthProvider;
 
     /* Firebase APIs */
 
     this.auth = app.auth();
-    this.db = app.database();
+
+    /* FireStore */
+    this.db = app.firestore();
+    const settings = { timestampsInSnapshots: true};
+    this.db.settings(settings);
```

Comment:
Maybe just inline the settings object here if it isn't used somewhere else.

Reply:
Yes it will be great